client/app/bundles/Room/components/Story.jsx

```diff
@@ -35,11 +35,23 @@ export default class Story extends React.Component {
       }
     })()
 
+    const storyTitle = (() => {
```

Comment:
This is a bugfix. Previously, story titles would get turned into links even if they weren't URLs. With this fix, only links that start with `"http"` will be linkified.